tests/test_nagbot.py

```diff
@@ -1,18 +1,19 @@
 import unittest
 
+from app import sqaws
 from app import nagbot
 from app.sqaws import Instance, Volume
 
 
-class TestNagbot(unittest.TestCase):
+class TestNewNagbot(unittest.TestCase):
```

Comment:
Four test cases fails for me when running test_nagbot file. I believe after the recent refactoring they may need minor tweaks.

Reply:
Yeah I totally forgot to update the testing files before committing - I've done that now and will be pushing those changes in a bit!